packages/maskbook/src/plugins/ITO/types.ts

```diff
@@ -49,6 +49,11 @@ export interface JSON_PayloadOutMask extends Omit<JSON_PayloadInMask, 'token' |
     exchange_tokens: TokenOutMask[]
 }
 
+export interface JSON_PayloadComposeMask extends Omit<JSON_PayloadInMask, 'token' | 'exchange_tokens'> {
```

Comment:
ITO payload sent by compose dialog. `token` and `exchange_tokens ` has been simplified.